### ISSUE TITLE
Coffeescript doesn't support #import

### ISSUE THREAD

**Issue by listrophy** [bendyworks/bwoken#1]
There's no way to get an #import statement into the generated javascript through pure coffeescript means... the compiler would choke.

I imagine having to use something like:

```
###
imports:
tuneup.js
###
```

which would result in:

```
#import "tuneup.js"
```

**Comment by listrophy** [comment]
Perhaps we can use [RequireJS](http://requirejs.org/). That would force users to replace

```
#import "tuneup.js"
test "foo", ->
  assertTrue true
```

with something like:

```
tuneup = require('tuneup.js')
tuneup.test 'foo', ->
  assertTrue true
```

**Comment by listrophy** [comment]
Or maybe just this?

```
require('tuneup')
test 'foo', ->
  assertTrue true
```

See [the API docs](http://requirejs.org/docs/api.html).

**Comment by listrophy** [comment]
Or perhaps even this:

```
require('tuneup.js')
test 'foo', ->
  assertTrue true
```

**Comment by Jaym3s** [comment]
Fixed in 52a4e705a8e1f6eab629c54cc5bb34925f7e2fce
